### ISSUE TITLE
psql ENUM recognised as user-defined

### ISSUE THREAD

**Issue by ghost** [danielnowak/sproc-spring-mapper#9]
In ObjectNode and ArrayNode (or similar) psql-enum's are recognised as user defined and treated as composite types, while they are a simple type (just text).

There is an easy workaround, that is converting the enum in text in the psql query, but it would be nice to use directly enums.

**Comment by hjacobs** [comment]
yes, we are aware of this problem and will solve this in the near future
